src/gui/qgsexpressionselectiondialog.cpp

```diff
@@ -42,7 +42,7 @@ QgsExpressionSelectionDialog::QgsExpressionSelectionDialog( QgsVectorLayer *laye
   connect( mButtonZoomToFeatures, &QToolButton::clicked, this, &QgsExpressionSelectionDialog::mButtonZoomToFeatures_clicked );
   connect( mPbnClose, &QPushButton::clicked, this, &QgsExpressionSelectionDialog::mPbnClose_clicked );
 
-  setWindowTitle( QStringLiteral( "Select by Expression - %1" ).arg( layer->name() ) );
+  setWindowTitle( QStringLiteral( "%1 - Select by Expression" ).arg( layer->name() ) );
```

Comment:
Typography alert: can you use an "em dash" here instead of a hyphen please?

Reply:
Done